source/tutorials/setap.rst

```diff
@@ -0,0 +1,373 @@
+
+* The Job can take long time to execute depending on the request and on the
+  platform load (typically form 2 to 20 hours).
+  When the processing is complete the status is set as "Successful Job"
+
+* Download the S1-ETAD products once the Job is completed.
+  Please note that the browsing layer of the S1-ETAD product, consisting in
+  a KMZ file, is also published separately, to allow a quicker download.
+
+.. figure:: assets/tuto_setap_6.png
+   :figclass: align-center
+   :width: 750px
+   :align: center
+
+-----
+
+.. _how-to-use-label:
+**How to use the S1-ETAD products**
+
+Basic information on the S1-ETAD product and how to use it are provided in
+the `Product Definition Document`_.
+Details on the file format and on the algorithms used to generaate it can
+be found in the `Format specification document`_ and the
+`Algorithm Theoretical Baseline Document`_ respectively.
+
+A `Python API`_ that allows to easily access the product and perform basic
+operations is also available.
+
+In general all main information and news about the S1-ETAD product are
+collected in the `Product Page`_ on the ESA web site.
+
+.. _`Product Definition Document`: https://sentinels.copernicus.eu/documents/247904/4629150/Sentinel-1-Product-Definition-Document.pdf
+.. _`Algorithm Theoretical Baseline Document`: https://sentinels.copernicus.eu/documents/247904/4629150/Sentinel-1-ETAD-Algorithm-Technical-Baseline-Document.pdf
+.. _`Python API`: https://gitlab.com/s1-etad/s1-etad
+.. _`Product Page`: https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1/data-products/etad-dataset
+
+.. _`sentinel-1-annotations-caching-policy-label`:
+**Sentinel-1 Annotations caching policy**
+
+The "Sentinel-1 Annotations” products (cf. "EO Data" menu) benefit from a
+data caching mechanism on GEP.
+The cache (Manifest and Annotation files) is managed for a pre-defined set
+of geographic areas and time span.
+These areas are specified according to the seismic areas defined by the
+`CEOS Seismic Hazards`_ work group, as follows:
+
+* mask_eu (Europe seismic areas)
+* mask1 (20% of he world's seismic areas)
+* ask2 (35%)
+* mask3 (50%)
+* mask70 (70%)
+* mask100 (100%)
+
+For the launch of the GEP SETAP service (September 2021), the cache is
+initialised as follows:
+
+* mask_eu (Europe)
+* January-September 2021
+* adding up new 2021 acquisitions from September 2021 onwards
+
+Moreover, when a processing Job is started over a spatial and temporal
+data search outside of the current data caching automation, an opportunistic
+approach will keep in cache the retrieved input products.
+
+This is resulting in an additional cache capacity, mutualised, that is
+growing based on user activities (so, without a planification on the GEP
+side, but with a monitoring of the user interests):
+
+* user selected job inputs falling outside current cached products,
+  for spatial and temporal domains with some limitations (see further
+  below the FAQ section, :ref:`Q1<faq-q1-label>_)
+
```

Comment:
@HerveCaumont Probably we could drop the bullet item formatting here

Reply:
Fine to me

Reply:
OK< thanks.
Done